commit 3aa7abd1983813b916e281a0e5940b5dd6768a9e
Author: Andrea Iacovitti <aiacovitti@libero.it>
Date:   Sat Sep 28 23:59:26 2013 +0200

    No need to compare in case-insensite way as adjustProtocol always return
    a lowercase string.

diff --git a/kio/kio/kprotocolmanager.cpp b/kio/kio/kprotocolmanager.cpp
index 05bb547..f1f356e 100644
--- a/kio/kio/kprotocolmanager.cpp
+++ b/kio/kio/kprotocolmanager.cpp
@@ -470,8 +470,7 @@ QStringList KProtocolManager::proxiesForUrl( const KUrl &url )
         const QString protocol = adjustProtocol(u.protocol());
         u.setProtocol(protocol);
 
-        if (protocol.startsWith(QL1S("http"), Qt::CaseInsensitive) ||
-            protocol.startsWith(QL1S("ftp"), Qt::CaseInsensitive)) {
+        if (protocol.startsWith(QL1S("http")) || protocol.startsWith(QL1S("ftp"))) {
           QDBusReply<QStringList> reply = QDBusInterface(QL1S("org.kde.kded"),
                                                          QL1S("/modules/proxyscout"),
                                                          QL1S("org.kde.KPAC.ProxyScout"))
